demisto_sdk/commands/common/content/content.py

```diff
@@ -74,17 +72,13 @@ def git() -> Repo | None:
         Notes:
             1. Should be called when cwd inside content repository.
         """
-        try:
-            if content_path := os.getenv("DEMISTO_SDK_CONTENT_PATH"):
-                repo = Repo(content_path)
-                logger.debug(f"Using content path: {content_path}")
-            else:
-                repo = Repo(Path.cwd(), search_parent_directories=True)
-        except InvalidGitRepositoryError:
-            logger.debug("Git repo was not found.")
-            repo = None
+        if content_path := os.getenv("DEMISTO_SDK_CONTENT_PATH"):
+            git_util = GitUtil(Path(content_path))
+            logger.debug(f"Using content path: {content_path}")
```

Comment:
```suggestion
```

let's move the log line into the `GitUtil` constructor, `f"GitUtil: using {content_path=}"`

Reply:
can't use logger in GitUtil due to circular import :(